cdap-docs/application-templates/source/etl/templates/sources/real-time/twitter.rst

```diff
@@ -6,9 +6,29 @@
 Sources: Real-time: Twitter 
 ===============================
 
-.. rubric:: Description: Twitter Realtime Source
+.. rubric:: Description
 
-Output records contain the fields *id* (long), *message* (string), *lang* (nullable
-string), *time* (nullable long), *favCount* (int), *rtCount* (int), *source* (nullable
+Samples tweets in realtime. Output records contain the fields *id* (long), *message* (string),
+*lang* (nullable string), *time* (nullable long), *favCount* (int), *rtCount* (int), *source* (nullable
```

Comment:
Notice that this file has the field names in italics (*). Looks nice, but we should either do it for all or none, right? What do you think?